ocs_ci/ocs/ui/base_ui.py

```diff
@@ -674,6 +676,82 @@ def navigate_block_pool_page(self):
         self.navigate_to_ocs_operator_page()
         self.do_click(locator=self.page_nav["block_pool_link"])
 
+    def navigate_cephblockpool(self):
```

Comment:
On PageNavigator class, we only to "main" pages on OCP console.. Please move these methods to anther class..

Reply:
done -> Separated to StorageSystemNavigator